CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ Changelog
 
 See also the [GitHub releases page](https://github.com/FriendsOfSymfony/FOSHttpCache/releases).
 
+2.15.4
+------
+
+* Add flag to `CustomTtlListener` for disabling fallback to s-maxage if header not defined.
+* Fix not calling store if Response object is not longer cacheable after event listeners.
```

Comment:
what exactly was the behaviour before? did it actually store, or is this only a performance optimization?

```suggestion
* Fix: Do not call store if Response object is not longer cacheable after event listeners.
```